src/features/customerView/queries.ts

```diff
@@ -76,6 +76,25 @@ export const INDIVIDUAL_CUSTOMER_QUERY = gql`
                 }
               }
             }
+            order {
+              dueDate
+              totalPrice
+              totalTaxPercentage
+              price
+              taxPercentage
+              status
+              orderLines {
+                edges {
+                  node {
+                    product {
+                      service
```

Comment:
```suggestion
                edges {
                  node {
                    id
                    product {
                      id
                      service
```

Reply:
I don't think this is relevant anymore, as the query now fetches `orders` instead of an `order` of a lease.